src/app/modules/item/components/add-dependency/add-dependency.component.html

```diff
@@ -0,0 +1,11 @@
+<alg-section icon="pi pi-plus" i18n-label label="Create a new dependency">
```

Comment:
In terms of UX, I think the search input should be cleared only when the dependency is properly added, not before. That way if anything goes wrong, the user can retry his search very easily.

Reply:
I'm totally agree. Probably we should have another issue as this component `AddContentComponent` is old and used in many places.

https://github.com/France-ioi/AlgoreaFrontend/blob/853750469f8906b5793f2ee30cbc5d154d0d29b9/src/app/modules/shared-components/components/add-content/add-content.component.ts#L95

Reply:
I've created the issue: https://github.com/France-ioi/AlgoreaFrontend/issues/1130